plugins/zenoh-plugin-storage-manager/src/replica/mod.rs

```diff
@@ -247,9 +247,12 @@ impl Replica {
                 .await;
             if to_be_processed {
                 log::trace!("[DIGEST_SUB] sending {} to aligner", digest.checksum);
-                match tx.send_async((from.to_string(), digest)).await {
+                match tx.try_send((from.to_string(), digest)) {
                     Ok(()) => {}
-                    Err(e) => log::error!("[DIGEST_SUB] Error sending digest to aligner: {}", e),
+                    Err(e) => {
+                        // Trace because this can happen _a lot_ on busy channels.
+                        log::trace!("[DIGEST_SUB] Error sending digest to aligner: {}", e)
+                    }
```

Comment:
I need to further investigate the implications of this change. As of today, I do not know if skipping digests could have unforeseen consequences on the replication (my first guess is that it doesn't but I want to make sure).

Reply:
Yup, it's not a small change. Ultimately this system needs to have some sort of backpressure and dropping because as a network of storages grows, if anything is out of sync it's impossible to parse and process _every_ digest w/o dropping.